update version to 1.0.1-SNAPSHOT, remove log4j and logback

sirolf2009 opened the pull request:
Fixes #8

iajn commented:
I don't think you fully understand what slf4j is and does. Its a log facade, basically just an API, that bridges logging to implementations (log4j, log4j2, logback). This way, users of your library are not forced to use the same logging implementation (which was the cause of the original issue).

To answer your points:
1. There is no "slf4j configuration". The log4j2.xml is no longer needed. Configuring the logger falls to the user of your library, and they configure it how they see fit for their application.

2. The output is caused by missing slf4j implementation. Adding one as a **test dependency** fixes the issue.

sirolf2009 commented:
sorry, I had completely missed your reply.
iajn is right, I've removed src/main/resources/pom.xml because it's unnecessary now that there isn't a log4j anymore. There's also nothing to port, because a library shouldn't come with its own logging config, the user should create his own. I've left src/test/resources/pom.xml and added log4j-slf4j-impl as a test dependency. So now when the tests are ran, it'll use a log4j implementation and not complain op NOP loggers.

martinfrancois commented on workbenchfx-core/pom.xml:
```diff
@@ -304,6 +304,17 @@
                         <goals>
                             <goal>shade</goal>
                         </goals>
+                        <configuration>
```
What is this change related to?

martinfrancois commented:
Since there was no answer from @sirolf2009, I reverted the unwanted change, now it's good to merge.
